src/less/chip/stories/static-chip.stories.js

```diff
@@ -0,0 +1,73 @@
+export default { title: "Skin/Chip/Static Chip" };
+
+export const typical = () => `
+<span class="chip">
+    <span class="chip__text">Football</span>
+</span>
+`;
+
+export const leastContent = () => `
+<span class="chip">
+    <span class="chip__text">I</span>
+</span>
+`;
+
+export const mostContent = () => `
+<span class="chip">
+    <span class="chip__text">Chip with the longest potentially possible content</span>
+</span>
+`;
+
+export const set = () => `
+<span class="chip">
+    <span class="chip__text">Football</span>
+</span>
+
+<span class="chip">
+    <span class="chip__text">Basketball</span>
+</span>
+
+<span class="chip">
+    <span class="chip__text">Baseball</span>
+</span>
+
+<span class="chip">
+    <span class="chip__text">Hockey</span>
+</span>
+
+<span class="chip">
+    <span class="chip__text">Tennis</span>
+</span>
+
+<span class="chip">
+    <span class="chip__text">Golf</span>
+</span>
+
+<span class="chip">
+    <span class="chip__text">Gymnastics</span>
+</span>
+
+<span class="chip">
+    <span class="chip__text">Swimming</span>
+</span>
+
+<span class="chip">
+    <span class="chip__text">Cricket</span>
+</span>
+
+<span class="chip">
+    <span class="chip__text">Volleyball</span>
+</span>
+
+<span class="chip">
+    <span class="chip__text">Rugby</span>
+</span>
+`;
+
+export const RTL = () => `
```

Comment:
I would like to see us get in the habit of adding stories for 200% text size and increased letter-spacing too (I forget what the WCAG threshold is for this one)

Reply:
Here it is: https://www.w3.org/WAI/WCAG22/Understanding/text-spacing

Reply:
I'll add the text size story. 

I'm not sure I understand the comment about letter spacing. Will chat offline.

Reply:
It's not something we've implemented yet, but text spacing test would be similar to the text-sizing 200% test. 

> Letter spacing (tracking) to at least 0.12 times the font size;

https://www.w3.org/WAI/WCAG22/Understanding/text-spacing
https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing